src/preloader/api.ts

```diff
@@ -13,19 +15,10 @@ export function createHttpServer(): MessagePort {
   return channel.port1;
 }
 
-export function createSocket(
-  host: string,
-  port: number
-): MessagePort | undefined {
+export function createSocket(host: string, port: number): MessagePort | undefined {
```

Comment:
should this be createTcpSocket?

Reply:
Ideally yes, but I didn't want to make this PR a breaking change. Maybe a follow-up PR that renames it and updates Studio?